lua/hop/hint.lua

```diff
@@ -15,11 +19,14 @@ end
 -- Regex hint mode.
 --
 -- Used to hint result of a search.
-function M.by_searching(pat, plain_search)
+function M.by_searching(pat, plain_search, direction, same_line, opts)
```

Comment:
Since this function is starting to get more arguments, I think it’s important to modify the documentation line 19-21 to explain what those are for.